structuredheadings/dialogs/selectstyle.js

```diff
@@ -1,13 +1,106 @@
+var selectedStyle;
+
+var columns = 4;
+var htmlContent = "";
+htmlContent += "<div>";
+htmlContent += "<table role=\"listbox\"";
+htmlContent += "style=\"width:100%;height:100%;border-collapse:separate;\"";
+htmlContent += "cellspacing=\"2\" cellpadding=\"2\"";
+if (CKEDITOR.env.ie && CKEDITOR.env.quirks) {
+  htmlContent += "style=\"position:absolute;\"";
+}
+htmlContent += "><tbody>";
```

Comment:
If in the right scope, you could then also test these individual functions rather than the entire thing (for example, test the correct row is generated for a given style, etc)